src/main.js

```diff
@@ -735,6 +735,8 @@ function showKills(simResult) {
     let dropsResultDiv = document.getElementById("simulationResultDrops");
     let newChildren = [];
     let newDropChildren = [];
+    let dropRateMultiplier = simResult.dropRateMultiplier;
+    let rareFindMultiplier = player.combatDetails.combatStats.combatRareFind + 1;
```

Comment:
In the same sense, although there are no rare find buffs currently, if you want to future-proof your code, you may want to do something similar to how you handle the drop rate multiplier.